planets/planetRegion.json

```diff
@@ -0,0 +1,354 @@
+{
+	"2632887550": {
+		"name": "DEMOCRACY ALWAYS",
+		"description": 0
```

Comment:
`description` is sometimes an integer (`0`) and sometimes a string
this will trip up any static typed language trying to handle them.
Can we either make them empty strings or omit the property entirely?